### PR TITLE
guide: Document how to enable PCP archiving

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: doc/guide/feature-pcp.xml [hunk 1]
```diff
@@ -11,6 +11,11 @@
     system from a different point in time. If PCP is not available, then Cockpit
     gathers the metrics data itself, but archival features are not available.</para>
 
+  <para>Whether or not metrics are archived depends on whether the
+    <code>pmlogger.service</code> and <code>pmcd.service</pmcd> systemd units
```
Comment (martinpitt) [inline doc/guide/feature-pcp.xml]: Should be `</code>` :)

File: doc/guide/feature-pcp.xml [hunk 2]
```diff
@@ -11,6 +11,11 @@
...
+    are running or not.  The "Store Performance Data" button in Cockpit
+    will enable and start these servivces.</para>
```
Comment (martinpitt) [inline doc/guide/feature-pcp.xml]: "servi*v*ces" typo

File: doc/guide/feature-pcp.xml (hunk 1 above)
Comment (mvollmer) [reply doc/guide/feature-pcp.xml]: Wow!

File: doc/guide/feature-pcp.xml (hunk 2 above)
Comment (mvollmer) [reply doc/guide/feature-pcp.xml]: Fixed

**Pull request merged**